src/android/src/main/java/io/callstack/react/fbads/BannerViewManager.java

```diff
@@ -37,10 +37,10 @@ protected BannerView createViewInstance(ThemedReactContext reactContext) {
   @Override
   public @Nullable Map getExportedCustomDirectEventTypeConstants() {
     return MapBuilder.of(
-      "onAdPress",
-      MapBuilder.of("registrationName", "onAdPress"),
-      "onAdError",
-      MapBuilder.of("registrationName", "onAdError")
+      "onPress",
+      MapBuilder.of("registrationName", "onPress"),
+      "onError",
+      MapBuilder.of("registrationName", "onError")
```

Comment:
I think there were issues with renaming them to onError and onPress. cc @jakubsta